data-types/objects/src/exercises/__tests__/1_assigning_values.test.js

```diff
@@ -0,0 +1,113 @@
+/*eslint-disable no-unused-expressions */
+`
+⬇️
+
+Welcome to Assigning Values
+
+This session is designed to introduce you to how assigning values to variables
+works in javascript. In javascript the typeof a value determines whether the
+variable is assigned by value or assigned by refererence.
+This exercise will discuss and demonstrate the implications of this difference.
+
+⬆️
+`;
+
+`📚 Exercise 1 - Assign by Value 📚
+
+In this exercise, we discuss the data types in javascript assigned by value.
+Numbers, Strings, Booleans, undefined, null, and Symbols are the different
+types of variables that are assigned by value. As a result, the variable's
+value can only be changed by reassigning the variable itself
+(The assigned value cannot be mutated).
+
+🛠️ Correct the test such that the expected asserted value is correct.
+
+💡 You should NOT modify the function for this exercise.
+💡 This exercise may seem super simple and obvious, but it serves as a
+   comparison to later exercises
```

Comment:
Nitpick: Need a 💡 here!

Reply:
hmmm but it is a continuation of the thought above, the equivalent would be:

- You should NOT modify the function for this exercise.
- This exercise may seem super simple and obvious, but it serves as a
- comparison to later exercises

vs 

- You should NOT modify the function for this exercise.
- This exercise may seem super simple and obvious, but it serves as a comparison to later exercises

Reply:
Sounds good!